src/workspaces/cookiefactoryv2/README.md

```diff
@@ -56,21 +104,59 @@ This project walks you through the process of setting up the Bakersville Cookie
       ```
     - Open scene in console: `https://us-east-1.console.aws.amazon.com/iottwinmaker/home?region=us-east-1#/workspaces/__FILL_IN__/scenes/CookieFactory`
 
-4. Setup Web Application
-   - Follow instructions in [CookieFactoryDemo](./CookieFactoryDemo/README.md)
+
+
+## Configure Web Application
+
+First change directory to CookieFactoryDemo
+``` 
+cd ../CookieFactoryDemo 
+```
+
+1. Change `WORKSPACE_ID` in `src/config/sites.ts` to your workspace ID.
+2. Use the following CLI command to administratively set the password for your demo Cognito user, Fran. Your User_Pool_id can be found after your CDK deploy completed in the CloudFormationOutput (printed to console). The demo Username is "fran@cookiefactory". Be sure to create a password that meets the default Cognito password requirements (Lowercase letter, Uppercase letter, Number, Symbol, Length >= 8)
+```
+aws cognito-idp admin-set-user-password --user-pool-id "[YOUR_USER_POOL_ID]" --username "[USERNAME]" --password "[PASSWORD]" --permanent
```

Comment:
maybe we can add `jq` as a dependency in the pre-reqs and then save the user-pool-id automatically for the user similar to in https://github.com/aws-samples/aws-iot-twinmaker-samples/tree/main/src/modules/insights

Reply:
I'll look into jq if I have some extra time, although I didn't see how it's used in that insights folder, is it in code or CLI commands?

Reply:
thanks, it's for the CLI, e.g. under "Set environment variables for convenience." in the README in insights folder

it's basically just a tool to let you extract/manipulate JSON, like setting an environment variable equal to a specific part of the output (e.g. the user pool id) which can be retrieved using AWS CLI to fetch the CFN output values